scss/base.scss

```diff
@@ -11,5 +11,11 @@ $govuk-global-styles: true;
     display: none;
 }
 
+mark {
+    background-color: inherit;
+    color: inherit;
+    font-weight: 700;
```

Comment:
Instead of hardcoding the 700, I think we can use a mixin from the design system here to ensure the font weight is consistent with any changes to that

https://frontend.design-system.service.gov.uk/sass-api-reference/#govuk-typography-weight-bold